idunn/blocks/commercial.py

```diff
@@ -0,0 +1,24 @@
+from .base import BaseBlock
+from typing import Literal, Optional
+
+
+class CommercialBlock(BaseBlock):
+    type: Literal["commercial"] = "commercial"
```

Comment:
I am a bit uncertain about this block name. An appointment may be relevant for non-commercial POIs (administration, health, etc.). Would `transactional` be better ?